libs/ledgerjs/packages/cryptoassets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ledgerhq/cryptoassets",
-  "version": "6.33.0",
+  "version": "6.34.0-next.0",
```

Comment:
this shouldn't happen in a PR.

Reply:
Probably caused by a bad merge. Fixed now.